docs/ru/query_language/table_functions/generate.md

```diff
@@ -0,0 +1,37 @@
+# generate
+
+Генерирует случайные данные с заданной схемой.
+Позволяет заполнять тестовые таблицы данными.
+Поддерживает все типы данных, которые могут храниться в таблице, за исключением `LowCardinality` и `AggregateFunction`.
+
+```sql
+generate('name TypeName[, name TypeName]...', 'limit'[, 'max_array_length'[, 'max_string_length'[, 'random_seed']]]);
+```
+
+**Входные параметры**
+- `name` — название соответствующего столбца.
+- `TypeName` — тип соответствующего столбца.
+- `limit` — количество строк для генерации.
+- `max_array_length` — максимальная длина массива для всех сгенерированных массивов. По умолчанию `10`.
+- `max_string_length` — максимальная длина строки для всех генерируемых строк. По умолчанию `10`.
+- `random_seed` — укажите состояние генератора случайных чисел вручную, чтобы получить стабильные результаты. По умолчанию `0` - генератор инициализируется случайным состоянием.
```

Comment:
Is it multithreaded?

Reply:
Current implementation is single threaded. Otherwise we should explicitly define number of threads in parameters for reproducible results.

Reply:
made it multithreaded